frontend/src/component/Activity/ActivityList.jsx

```diff
@@ -130,22 +130,26 @@ const ActivityList = (props) => {
 
   const isNotPublicationOrInWorkspaceWithActivatedPublications = (activity) => {
     if (activity.content.content_namespace !== CONTENT_NAMESPACE.PUBLICATION ||
-        !activity.newestMessage.fields.workspace) return true
+      !activity.newestMessage.fields.workspace) return true
     const activityWorkspace = props.workspaceList.find(ws => ws.id === activity.newestMessage.fields.workspace.workspace_id)
     if (!activityWorkspace) return true
     return activityWorkspace.publicationEnabled
   }
 
   const isLoggedUserMember = (activity) => props.workspaceList.find(space => space.id === activity.newestMessage.fields.workspace.workspace_id)
 
+  const isAttachedFileOnPublication = (activity) => activity.content
```

Comment:
isn't "publication" the french translation for "news"? I think the feature regarding code should be named "activity", like you did in the parameter

Reply:
actually, the french translation is "Actualité"
but all theses words point to the same feature. What is the official word to use in the code, activity or publication?

Reply:
Explained irl, I will update the variable to `isActivityAnAttachedFileOnPublication`